contribute/design/1.x/ndm/20190705-ndm-disk-identification.md

```diff
@@ -0,0 +1,184 @@
+---
+oep-number: NDM 0001
+title: Uniquely identify disk in cluster
+authors:
+  - "@akhilerm"
+owners:
+  - "@gila"
+  - "@kmova"
+  - "@vishnuitta"
+editor: "@akhilerm"
+creation-date: 2019-07-05
+last-updated: 2019-07-08
+status: provisional
+---
+
+# Disk Identification using NDM
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Stories [optional]](#user-stories-optional)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+    * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Drawbacks [optional]](#drawbacks-optional)
+* [Alternatives [optional]](#alternatives-optional)
+* [Infrastructure Needed [optional]](#infrastructure-needed-optional)
+
+## Summary
+
+This proposal brings out the design details to implement a Disk identification
+solution which can be used to uniquely identify a disk in the cluster. The main
+aim is to identify the data.
```

Comment:
The last part does not make sense to me. We are clearly speaking about disks here. Let us keep it at that?